tests/visual/Text.js

```diff
@@ -0,0 +1,20 @@
+context('Text', () => {
+   beforeEach(() => {
+      cy.viewport(500, 500)
+   })
+
+   it('Renders at different sizes as expected', () => {
```

Comment:
Might wanna just clarify that the `sizes` applies to text size, and not window size or something